binary_transparency/firmware/cmd/ft_monitor/main.go

```diff
@@ -116,13 +121,27 @@ func main() {
 				glog.Warningf("Unable to GetFirmwareImage for Firmware with Hash 0x%x , reason %q", meta.FirmwareImageSHA512, err)
 				continue
 			}
+
 			// Verify Image Hash from log Manifest matches the actual image hash
 			h := sha512.Sum512(image)
 			if !bytes.Equal(h[:], meta.FirmwareImageSHA512) {
 				glog.Warningf("downloaded image does not match SHA512 in metadata (%x != %x)", h[:], meta.FirmwareImageSHA512)
 				continue
 			}
 			glog.V(1).Infof("Image Hash Verified for image at leaf index %d", manifest.LeafIndex)
+
```

Comment:
Travis integration test is complaining about formatting:

```
binary_transparency/firmware/cmd/ft_monitor/main.go:133: File is not `goimports`-ed (goimports)
```

Reply:
Thanks, will check, what is the issue here!